modules/oadp-installing-oadp-rosa-sts.adoc

```diff
@@ -15,9 +15,128 @@ Restic is not supported in the OADP on ROSA with AWS STS environment. Ensure the
 
 .Prerequisites
 
-* A ROSA OpenShift Cluster with the required access and tokens.
+* A ROSA OpenShift Cluster with the required access and tokens. See procedure in *Preparing AWS account* below.
 * link:https://docs.openshift.com/container-platform/4.13/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.html#oadp-creating-default-secret_installing-oadp-aws[A default Secret], if your backup and snapshot locations use the same credentials, or if you do not require a snapshot location.
 
+.Preparing AWS account
```

Comment:
This section should be placed in a separate module with section `Procedure`. You could place this module separately either before or after this module,

Reply:
applied, thanks

Reply:
PM @arendej wants this section to remain within the module and not separate, thanks